docs/Reference/API-Objects.md

```diff
@@ -193,10 +193,14 @@ Returned by [eth_getTransactionReceipt](API-Methods.md#eth_gettransactionreceipt
 
 Returned by [priv_getTransactionReceipt](API-Methods.md#priv_getTransactionReceipt).
 
-| Key                   | Type                 | Value                                                                                |
-|-----------------------|:--------------------:|--------------------------------------------------------------------------------------|
-| **contractAddress**   | Data, 20&nbsp;bytes  | Contract address created, if contract creation transaction; otherwise, `null`.       |
-| **from**              | Data, 20&nbsp;bytes  | Address of the sender.                                                               |
-| **to**                | Data, 20&nbsp;bytes  | Address of the receiver, if sending ether; otherwise, null.                          |
-| **output**            | Data                 | RLP-encoded return value of a contract call, if value is returned; otherwise, `null`.|
-| **logs**              | Array                | Array of [log objects](#log-object) generated by this transaction.                   |
\ No newline at end of file
+| Key                                  | Type                         | Value                                                                                |
+|-----------------------               |:--------------------:        |--------------------------------------------------------------------------------------|
+| **contractAddress**                  | Data, 20&nbsp;bytes          | Contract address created, if contract creation transaction; otherwise, `null`.       |
+| **from**                             | Data, 20&nbsp;bytes          | Address of the sender.                                                               |
+| **output**                           | Data                         | RLP-encoded return value of a contract call, if value is returned; otherwise, `null`.
+| **commitmentHash**                   | Data, 32&nbsp;bytes          | Hash of the privacy marker transaction                          |
+| **transactionHash**                  | Data, 32&nbsp;bytes          | Hash of private transaction
+| **privateFrom**                      | Data, 32&nbsp;bytes          | [Orion](https://docs.orion.pegasys.tech/en/stable/) public key of sender                                                      |
+| **privateFor** or **privacyGroupId** | Array or Data, 32&nbsp;bytes | [Orion](https://docs.orion.pegasys.tech/en/stable/) public keys or privacy group ID of recipients  
```

Comment:
needs to be double checked

Reply:
What needs to be double checked?

Reply:
so this will depend on how the original private transaction was sent. If it was sent with privateFrom/privateFor then the receipt will be missing the privacyGroupId. Similarly if it was sent with privateFrom/privacyGroupId the receipt will be missing privateFor